dataprocspawner/spawner.py

```diff
@@ -25,6 +25,7 @@
 from google.cloud.dataproc import ClusterControllerClient
```

Comment:
Should we change all of these to be google.cloud.dataproc_v1beta2.<class>? I'm wondering if we're kind of mixing between these two.

Reply:
Updated in commit bac00af